packages/did-ethr-resolver/test/did-operator.test.ts

```diff
@@ -38,12 +39,32 @@ const validity = 10 * 60 * 1000;
 const did = `did:ethr:${identity}`;
 let operator: Operator;
 let registry: string;
+let registrySettings: RegistrySettings;
 
 const testSuite = (): void => {
   it('operator public key should be equal to public key of signer', () => {
     expect(operator.getPublicKey()).equal(keys.publicKey);
   });
 
+  it('operator should use provider from signer', async () => {
```

Comment:
Hmm, for me comparing number of providers set on different operator is quite indirect way to ensure that signer and operator are connected through the same provider.  I suggest to mock signer's provider, then invoke some method on operator and verify that corresponding method on mocked provider has been invoked

Reply:
That's a good suggestion, thank you. I implemented this using a `spy`, let me know what you think.